src/data_management/rasters/DeleteRasterSourceNotAllowed.tsx

```diff
@@ -20,32 +21,32 @@ const DeleteRasterSourceNotAllowed: React.FC<MyProps> = (props) => {
   // const fetchLayers = (uuids: string[],) => {
   //   const baseUrl = "/api/v4/rasters/";
   //   const fetches = uuids.map (uuid => {
-  //     return (fetch(baseUrl + uuid + "/").then((res:any)=>res.json()));
+  //     return (fetch(baseUrl + uuid + "/").then((res)=>res.json()));
   //   });
   //   return Promise.all(fetches)
   // }
   
```

Comment:
While we're doing this, could we also remove commented out code?

Reply:
Removed